site/content/docs/statistics/highest-false-positives-last-month.md

```diff
@@ -0,0 +1,10 @@
+---
+type: page
+title: Highest False Positives Last Month
+---
+
+These were the alerts most frequently flagged as false positives using [Alert Filters](/docs/desktop/addons/alert-filters/) last month.
```

Comment:
Have we added stats to core to track when people manually set FP? Because that would also be good to know (separately).

Reply:
I'll check - if not then yes, we should...